src/openrct2/actions/LargeSceneryRemoveAction.cpp

```diff
@@ -191,21 +191,18 @@ GameActions::Result::Ptr LargeSceneryRemoveAction::Execute() const
 
 TileElement* LargeSceneryRemoveAction::FindLargeSceneryElement(const CoordsXYZ& pos, int32_t sequenceIndex) const
 {
+    const bool isGhost = GetFlags() & GAME_COMMAND_FLAG_GHOST;
     for (auto* sceneryElement : TileElementsView<LargeSceneryElement>(pos))
     {
         // If we are removing ghost elements
-        if ((GetFlags() & GAME_COMMAND_FLAG_GHOST) && sceneryElement->IsGhost() == false)
+        if (isGhost && sceneryElement->IsGhost() == false)
             continue;
-
```

Comment:
Please keep this blank lines, they help readability.

Reply:
Its less readable the other way imho.

Reply:
I agree with @Gymnasiast and it's the general style used in the rest of the codebase for ifs without curly brackets

Reply:
The consistency part is actually not true, see:
https://github.com/OpenRCT2/OpenRCT2/blob/14cca95fecdb48f88e51e2f372eed5bf545ec022/src/openrct2/actions/BannerRemoveAction.cpp#L128
https://github.com/OpenRCT2/OpenRCT2/blob/14cca95fecdb48f88e51e2f372eed5bf545ec022/src/openrct2/actions/FootpathPlaceAction.cpp#L439
https://github.com/OpenRCT2/OpenRCT2/blob/14cca95fecdb48f88e51e2f372eed5bf545ec022/src/openrct2/actions/LandSetHeightAction.cpp#L254
etc..

Its inconsistent, thats the whole reason why I removed those empty lines, but I added them back.